internal/staging/stdscript/scriptv0_test.go

```diff
@@ -1235,6 +1235,85 @@ func TestExtractTreasuryGenScriptHashV0(t *testing.T) {
 	}
 }
 
+// TestProvablyPruneableScriptV0 ensures generating a version 0
+// provably-pruneable nulldata script works as intended for
```

Comment:
It looks like your comment was cut short here.

Reply:
I meant to fix it in this PR, but looks like I fixed it later.  I've put the fix in the appropriate commit now.